### PR TITLE
MGMT-4100 Move download logs logic to download_logs.sh script

### PR DESCRIPTION
**Pull request by YuviGold**
Support onprem and k8s logs with getting `ADDITIONAL_PARAMS` environment variable. i.e. `--must-gather`, `--download-all`, `--update-by-events`.

/cc @osherdp

### CODE REVIEW

File: discovery-infra/logger.py [hunk 1]
```diff
@@ -17,6 +18,16 @@ def format(self, record):
         return self._filter(original)
 
 
+class suppressLog(suppress):
```
Comment (osherdp) [inline discovery-infra/logger.py:21]: it's a misleading name, I thought it just suppresses logging inside the block
Maybe ``suppressAndLog`` or ``suppressButLog``? Or ``suppressWithNotification``/``WithLogging``

**Review by osherdp — CHANGES_REQUESTED** [review]
just the class name sounds weird

**Comment by YuviGold** [comment]
I divided this PR into another PR https://github.com/openshift/assisted-test-infra/pull/524

**Review by osherdp — APPROVED** [review]
/lgtm

**Comment by osherdp** [comment]
@YuviGold all latest failures:
```
HTTP response body: {"code":"500","href":"","id":500,"kind":"Error","reason":"coreos-installer failed: /data/coreos-installer: error while loading shared libraries: libssl.so.1.1: cannot open shared object file: No such file or directory\n: exit status 127"}
```

Not relevant at all to your changes, which makes me think it's somehow got broken

**Pull request merged**